src/NzbDrone.Core/Indexers/Torznab/TorznabRssParser.cs

```diff
@@ -99,6 +100,19 @@ protected override string GetDownloadUrl(XElement item)
             return url;
         }
 
+        protected virtual Int32 GetTvdbId(XElement item)
```

Comment:
Not in this file, I tend to keep a file consistent until we can get Roslyn to replace em all.